app/controllers/pets_controller.rb

```diff
@@ -1,10 +1,13 @@
 class PetsController < ApplicationController
+  skip_before_action :authenticate_user!, only: %i[index show]
+  before_action :set_pet, only: %i[show edit update destroy]
+
   def index
     @pets = Pet.all
   end
 
   def show
-    @pet = Pet.find(params[:id])
+    @review = Review.new
```

Comment:
Why are we creating a review in the pet controller? I think we are mixing if we don something like this

Reply:
Yeah similar to what pablo said shouldn't we leave the review.new to the review controller and then just iterate over bookings.reviews to display in the index page

Reply:
Initially I thought we should show the review in the booking, so we need to have the @pet and @review in the booking so that we're able to show them on the same page.